packages/storybook/stories/Overview.stories.mdx

```diff
@@ -64,109 +64,43 @@ Azure Communication Services support UI Library across Web (React) and Native Mo
 npm i @azure/communication-react
 ```
 
-## Composites Overview
-
-Composites are higher-level components composed of UI components that deliver turn-key solutions for common communication scenarios using Azure Communication Services.
-Developers can easily instantiate the Composite using an Azure Communication Services access token and the required configuration attributed for call or chat.
-
-### Call composite
-
-[CallComposite](./?path=/story/composites-call-basicexample--basic-example) is providing a calling experience that allows users to start or join a call. Inside the experience users can configure their
-devices, participate in the call with video and see other participants, including those with video turn on.
-
-Here are visual examples of Call composite where participants have their camera turned off, then some ahve them turned on.
-People speaking with microphone unmuted will have their tile appearing with a blue border.
-
-<Stack horizontal horizontalAlign="space-between" tokens={{ childrenGap: '2rem' }} style={overviewPageImagesStackStyle}>
-  <Stack.Item align="center">
-    <img
-      src="images/callComposite_CamerasOff.png"
-      alt="Call composite where all participants have their camera turned off"
-    />
-  </Stack.Item>
-  <Stack.Item align="center">
-    <img
-      src="images/callComposite_CamerasOn.png"
-      alt="Call composite where some participants have their camera turned on"
-    />
-  </Stack.Item>
-</Stack>
-
-When joining a Microsoft Teams call ([Teams Interop experience](./?path=/docs/examples-teamsinterop-compliancebanner--compliance-banner)), the
-Call Composite includes a lobby experience where the user is waiting to be admitted into the call.
+## UI Artifacts
 
 <Stack horizontal horizontalAlign="space-between" tokens={{ childrenGap: '3rem' }} style={overviewPageImagesStackStyle}>
```

Comment:
Consider a wrapping property to prevent items becoming too small to fit text:
![image](https://user-images.githubusercontent.com/2684369/164082079-186b2bb1-54bc-4d67-a7d7-bbd378bb58b9.png)

Reply:
Resolved this with newer design